README.md

```diff
@@ -36,7 +39,7 @@ console.log(third_array) // [4, 8, 3]
 ```
 
 ```js
-function reverse(array) {
-  // -- YOUR ANSWER HERE --
+function swap(arr3) {
+  console.log(arr3.reverse())
```

Comment:
While you do want to use the reverse function, reverse is not a method of the arr3 variable.